docs/tutorials/how-to-estimator.ipynb

```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a4477909",
+   "metadata": {},
+   "source": [
+    "# Get started with the Estimator primitive\n",
+    "\n",
+    "Learn how to set up and use the Estimator primitive program.\n",
+    "\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "The Estimator primitive lets you efficiently calculate and interpret expectation values of quantum operators required for many algorithms. You can specify a list of circuits and observables, then evaluate expectation values and variances for a given parameter input.  \n",
+    "\n",
+    "\n",
+    "## Prepare the environment\n",
+    "\n",
+    "1. Follow the steps in the [getting started guide](/docs/quantum-computing?topic=quantum-computing-quickstart) to get your quantum service instance ready to use.\n",
```

Comment:
This link is broken. I think it points to an IBM Cloud specific doc. Should this point to a channel agnostic doc on qiskit.org on how to get started or should it point to both IBM Cloud and IBM Quantum docs?

Reply:
@rathishcholarajan , @frankharkins , I needed this jupyter nb so [I fixed this on my PR](https://github.com/Qiskit/qiskit-ibm-runtime/pull/263/commits/177d1f3dc591fd7dda5eb8766f6311520901a437). If you both agree, we can merge the changes on that PR instead of using this one